lib/states/statesInMemClientSocketIo.js

```diff
@@ -0,0 +1,380 @@
+/**
+ *      States DB in memory - Client
+ *
+ *      Copyright 2013-2018 bluefox <dogafox@gmail.com>
+ *
+ *      MIT License
+ *
+ */
+
+/** @module statesRedis */
+
+/* jshint -W097 */
+/* jshint strict: false */
+/* jslint node: true */
+'use strict';
+
+const io = require('socket.io-client');
+
+/** @class */
+function StatesInMemClientSocketIo(settings) {
```

Comment:
Is this copied from the old code or do I need to review this?

Reply:
This is copy of the old file ... so no review needed